backend/src/main/kotlin/no/uib/echo/DatabaseHandler.kt

```diff
@@ -173,6 +173,7 @@ class DatabaseHandler(
             "test.mctest@student.uib.no",
             "Test McTest",
             memberships = listOf("webkom"),
+            dots = 0,
```

Comment:
Kan vi kalle det `strikes` i stedet? overalt
```suggestion
            strikes = 0,
```